ironfish-cli/src/commands/wallet/mint.ts

```diff
@@ -339,4 +342,29 @@ Find the transaction on https://explorer.ironfish.network/transaction/${transact
       this.exit(2)
     }
   }
+
+  async doElegibilityCheck(): Promise<void> {
+    const api = new WebApi()
+    const graffiti = this.sdk.config.get('blockGraffiti')
+    const user = await api.findUser({ graffiti: graffiti })
+    if (!user) {
+      this.log(`WARNING: Could not find a user with graffiti ${graffiti}`)
```

Comment:
If they have not set a graffiti yet it just prints

```bash
$ ironfish wallet:mint

WARNING: Could not find a user with graffiti
Do you want to create a new asset (Y/N)?:
```
It's a bit weird, should we avoid doing this or print a different message if they have no block graffiti?